### PR TITLE
ParseEmailFiles Polish lang - decode fix

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35072

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by anara123 — APPROVED** [review]

File: Packs/CommonScripts/ReleaseNotes/1_3_31.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### ParseEmailFiles
+- Fixed an issue where the script failed to parse email containing Polish.
```
Comment (amshamah419) [inline Packs/CommonScripts/ReleaseNotes/1_3_31.md:4]:
```suggestion
- Fixed an issue where the script failed to parse an email containing Polish characters.
```

**Comment by guykeller** [comment]
@ChanochShayner Is adding a unit test here feasible in your opinion? If it isn't, why not?

**Comment by ChanochShayner** [comment]
> @ChanochShayner Is adding a unit test here feasible in your opinion? If it isn't, why not?

If we have an email to use it for a test it will be great - waiting for an answer from the customer about it

**Comment by guykeller** [comment]
@ChanochShayner Just make sure to very carefully remove all sensitive information from that email before pushing it to the PR. You can consult with me once you get it.

**Comment by ChanochShayner** [comment]
@guykeller UT added (possible even without the email)
image
image

**Pull request merged**